product_serial/i18n/de.po

```diff
@@ -0,0 +1,221 @@
+msgid "Last Number"
+msgstr "Letzte Nummer"
+
+#. module: product_serial
+#: selection:product.product,lot_split_type:0
+msgid "Single"
+msgstr "Einzel"
+
+#. module: product_serial
+#: help:stock.production.lot,last_location_id:0
+msgid "Display the current stock location of this production lot"
+msgstr "Aktuellen Lagerplatz dieses Fertigungsloses anzeigen"
+
+#. module: product_serial
+#: selection:product.product,lot_split_type:0
+msgid "None"
+msgstr "Kein"
+
+#. module: product_serial
+#: model:ir.model,name:product_serial.model_stock_picking_prodlot_selection
+msgid "stock.picking.prodlot.selection"
+msgstr "stock.picking.prodlot.selection"
+
+#. module: product_serial
+#: field:stock.production.lot,last_location_id:0
+msgid "Last location"
+msgstr "Letzter Platz"
+
+#. module: product_serial
+#: view:stock.move:0
+msgid "[('product_id','=?',product_id)]"
+msgstr "[('product_id','=?',product_id)]"
+
+#. module: product_serial
+#: view:stock.move:0
+msgid "Manual Split"
+msgstr "Manuelle Aufteilung"
+
+#. module: product_serial
+#: model:ir.model,name:product_serial.model_stock_production_lot
+msgid "Serial Number"
+msgstr "Seriennummer"
+
+#. module: product_serial
+#: model:ir.model,name:product_serial.model_stock_picking
+msgid "Picking List"
+msgstr "Kommissionierschein"
+
+#. module: product_serial
+#: field:stock.move,new_tracking_code:0
+msgid "Create Tracking"
+msgstr "Trackingnummer anlegen"
+
+#. module: product_serial
+#: field:stock.picking.prodlot.selection,create_prodlots:0
+msgid "Create New Serial Numbers"
+msgstr "Neue Seriennummern anlegen"
+
+#. module: product_serial
+#: field:stock.picking.prodlot.selection,first_number:0
+msgid "First Number"
+msgstr "Erste Nummer"
+
+#. module: product_serial
+#: model:ir.model,name:product_serial.model_product_product
+#: field:stock.picking.prodlot.selection,product_id:0
+msgid "Product"
+msgstr "Produkt"
+
+#. module: product_serial
+#: view:stock.move:0
+msgid "bottom"
+msgstr "unten"
+
+#. module: product_serial
+#: field:stock.picking.prodlot.selection,prefix:0
+msgid "Prefix"
+msgstr "Präfix"
+
+#. module: product_serial
+#: field:stock.move,new_prodlot_code:0
+msgid "Create Serial Number"
+msgstr "Seriennummer anlegen"
+
+#. module: product_serial
+#: field:res.company,autosplit_is_active:0
+msgid "Active auto split"
+msgstr "Auto-Split aktiv"
+
+#. module: product_serial
+#: field:res.company,is_group_invoice_line:0
+msgid "Group invoice lines"
+msgstr "Rechnungszeilen gruppieren"
+
+#. module: product_serial
+#: model:ir.actions.act_window,name:product_serial.action_prodlot_selection
+#: view:stock.picking.prodlot.selection:0
+msgid "Select Production Lots"
+msgstr "Fertigungslose auswählen"
+
+#. module: product_serial
+#: view:stock.picking.prodlot.selection:0
+msgid "Validate"
+msgstr "Prüfen"
+
+#. module: product_serial
+#: view:stock.move:0
+msgid ""
+"onchange_lot_id(prodlot_id, product_qty, location_id, product_id, "
```

Comment:
Yeah, that's it. Export tool is taking this into account because it takes string values and put it here. It doesn't filter about what values to export (or it doesn't make it right).